docs/installation/osx.rst

```diff
@@ -9,9 +9,6 @@ If you are running OS X, you can install everything needed with Homebrew.
 
        xcode-select --install
 
-#. Install `XQuartz <http://xquartz.macosforge.org/>`_. This is needed by
-   GStreamer which Mopidy use heavily.
```

Comment:
I have no idea if this is still required. I assume not. Have you tried installing Mopidy on a clean macOS system recently?

Reply:
Yes I installed Mopidy on a clean instance of OSX High Sierrra (10.13.16) and got it running by just following the Homebrew steps.